docker/advanced_settings.md

```diff
@@ -307,6 +307,7 @@ Notes on a couple of the parameters:
 * **keystore_type** - Store of cryptographic keys and certificates
 * **private_key_file** - Location of the private key file
 * **certificate_file** - Location of the certificate file
+* **backend_response_buffer_size** - The maximum buffer size the frontend allocates for a worker response, in bytes.
```

Comment:
Can we change it to max_response_size?

backend is a bit ambiguous.

Reply:
Should we split them to 2 attributes then (`max_backend_response_size` and `max_response_size`) or just define a single `max_response_size` which both the frontend and the backend will share?

Reply:
I also recommend not using "backend" in the configuration parameters. Its better to leave it as "max-message-size" or something generic like that. I see value in having separate configuration parameters for backend request and response, from memory point of view, but I also feel we are adding too many configuration parameters. I would recommend having only one parameter for now and later on, based on requirements, we could add more.

Reply:
Sounds reasonable, renamed to `max_response_size`.